### PR TITLE
MOD ID added

### PR DESCRIPTION
**Pull request by Natclanwy**

### CODE REVIEW

**Comment by Bletch1971** [comment]
Need to remove the [ ] from the Mod field.

File: GameData/Maps/Genesis2.gamedata [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "Application": "ark",
-  "Version": "2.1.5",
-  "Created": "2021-07-05T08:35:00.0000000Z",
+  "Version": "2.1.6",
+  "Created": "2021-12-12T18:00:00.0000000Z",
+  "Mod": "[Genesis2]",
```
Comment (Bletch1971) [inline GameData/Maps/Genesis2.gamedata]: remove the []

File: GameData/Mods/ArkOmega.gamedata [hunk 2]
```diff
@@ -0,0 +1,3552 @@
+{
+  "Application": "ark",
+  "Version": "2.3.2",
+  "Created": "2021-12-12T00:00:00.0000000Z",
+  "Mod": "[2447186973]",
```
Comment (Bletch1971) [inline GameData/Mods/ArkOmega.gamedata]: Remove the []

File: GameData/Mods/SuperSpyglass.gamedata [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
   "Application": "ark",
-  "Version": "1.0.0",
-  "Created": "2021-06-07T20:10:00.0000000Z",
+  "Version": "1.0.1",
+  "Created": "2021-12-12T18:00:00.0000000Z",
+  "Mod": "[793605978]",
```
Comment (Bletch1971) [inline GameData/Mods/SuperSpyglass.gamedata]: Remove the []

File: GameData/Template/Template.gamedata [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "Application": "ark",
   "Version": "1.0.0",
   "Created": "2020-03-04T00:00:00.0000000Z",
-  "Mod": "[ArkPrime|TheCenter|PrimitivePlus|ScorchedEarth|Ragnarok|Aberration|Extinction|Genesis|CrystalIsles|Genesis2|LostIsland|<modid>]"
+  "Mod": "[ArkPrime|TheCenter|PrimitivePlus|ScorchedEarth|Ragnarok|Aberration|Extinction|Genesis|CrystalIsles|Genesis2|LostIsland|<modid>]",
```
Comment (Bletch1971) [inline GameData/Template/Template.gamedata]: Maybe remove the [] from the Mod field. So no one gets confused.

File: GameData/Mods/Archived/Super_Spyglass_20210813.gamedata [hunk 5]
```diff
@@ -1,21 +1,25 @@
 {
   "Application": "ark",
-  "Version": "1.0.0",
-  "Created": "2021-06-07T20:10:00.0000000Z",
+  "Version": "1.0.1",
+  "Created": "2021-12-12T18:00:00.0000000Z",
+  "Mod": "[793605978]",
```
Comment (Bletch1971) [inline GameData/Mods/Archived/Super_Spyglass_20210813.gamedata:5]: found some []

File: GameData/Mods/LostIslandDinoEngramm.gamedata [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{
+  "Application": "ark",
+  "Version": "1.0.0",
+  "Created": "2021-12-15T00:00:00.0000000Z",
```
Comment (Bletch1971) [inline GameData/Mods/LostIslandDinoEngramm.gamedata]: No mod field

**Review by Bletch1971 — COMMENTED** [review]
Such a great job - thank you.
Just a few I found.

File: GameData/Mods/Archived/Super_Spyglass_20210813.gamedata (hunk 5 above)
Comment (Natclanwy) [reply GameData/Mods/Archived/Super_Spyglass_20210813.gamedata:5]: This one is archived and the correct one is in the Mod Folder.

Comment (Natclanwy) [reply GameData/Mods/Archived/Super_Spyglass_20210813.gamedata:5]: This on had a date in the file name so I just left the one without a date.

File: GameData/Mods/LostIslandDinoEngramm.gamedata (hunk 6 above)
Comment (Natclanwy) [reply GameData/Mods/LostIslandDinoEngramm.gamedata]: I'll add the mod field.

**Pull request merged**